src/hb-ot-color.h

```diff
@@ -50,7 +50,7 @@ HB_BEGIN_DECLS
  */
 typedef struct
 {
-  uint8_t red, green, blue, alpha;
+  uint8_t alpha, red, green, blue;
```

Comment:
I'm now reminded that we cannot have a union of int and four uint8_t's as that would become endianness-dependent. So, I think we want it as a uint32_t. Kinda like a hb_tag_t. Essentially you treat the color as an integer.

Reply:
So the removal hb_{ot_}color_t? To be honest I like that!